src/javascript/binary/pages/trade/message.js

```diff
@@ -27,14 +27,34 @@ var Message = (function () {
                 processTick(response);
             } else if (type === 'trading_times'){
                 processTradingTimes(response);
+            } else if (type === 'statement'){
+                StatementWS.statementHandler(response);
+            } else if (type === 'balance'){
+                var passthroughObj = response.echo_req.passthrough;
```

Comment:
I know you guys use the passthrough in several places already, but not fan of it.
I think it is a bit of an anti-pattern, and have managed to do all my coding without it (just using it for ID).
(just sharing thoughts, nothing to fix ;) )